tests/conftest.py

```diff
@@ -2333,6 +2333,30 @@ def titanic_pandas_data_context_with_v013_datasource_with_checkpoints_v1_with_em
         """
 
     context.test_yaml_config(name="my_datasource", yaml_config=datasource_config)
+
```

Comment:
I'm seeing a number of tests that use this feature are currently failing... would it help to split it out into two sets of fixtures? 
ie. one that lives in `tests/data_context/conf.py` and another that lives in `tests/checkpoint/conf.py`?

Reply:
Just fixed this!  We now have an SQL Execution Engine in one of the fixtures -- this required a more careful checking.

Reply:
Ok wonderful to hear @alexsherstinsky  :) Will defer to @jcampbell for final approval